iron-collapse.html

```diff
@@ -239,6 +239,14 @@
       this.notifyResize();
     },
 
+    _isVisible: function() {
```

Comment:
Since this method tells us if is displayed, can we name it `_isDisplayed`?
Also, any reason why making this a method and not a getter?
Finally, can you add a jsdoc for it & mark it `@private`?